src/Controller.php

```diff
@@ -63,17 +70,100 @@ public function connect(Application $app)
      */
     public function thumbnail(Application $app, Request $request, $file, $action, $width, $height)
     {
+        // Set to default 404 image if restricted to aliases
+        if($this->isRestricted($app, $request)) {
+            $transaction = $this->defaultTransaction($app, $request);
+            $thumbnail = $app['thumbnails']->respond($transaction);
+            return new Response($thumbnail);
+        }
+
+        return $this->serve($app, $request, $file, $action, $width, $height);
+    }
+
+    /**
+     * Returns a thumbnail response.
+     *
+     * @param Application $app
+     * @param Request     $request
+     * @param string      $file
+     * @param string      $alias
+     *
+     * @return Response
+     */
+    public function alias(Application $app, Request $request, $file, $alias)
+    {
+        $config = $app["config"]->get("theme/image_aliases/".$alias, false);
+
+        // Set to default 404 image if alias does not exist
+        if(!$config) {
+            $transaction = $this->defaultTransaction($app, $request);
+            $thumbnail = $app['thumbnails']->respond($transaction);
+            return new Response($thumbnail);
+        }
+
+        $width  = isset($config["size"][0])  ? $config["size"][0]  : 0;
+        $height = isset($config["size"][1])  ? $config["size"][1]  : 0;
+        $action = isset($config["cropping"]) ? $config["cropping"] : Action::CROP;
```

Comment:
Only the config uses `cropping`. See here:
```
thumbnails:
    default_thumbnail: [ 160, 120 ]
    default_image: [ 1000, 750 ]
    quality: 80
    cropping: crop
```
I'll change back to action anyways. Seems more consistent.